web/gds-user-ui/src/components/TrisaImplementationForm/index.tsx

```diff
@@ -48,15 +64,7 @@ const TrisaImplementationForm: React.FC<TrisaImplementationFormProps> = ({ heade
         label="Certificate Common Name"
         placeholder="trisa.example.com"
```

Comment:
Nice job @elysee15. Could you replace the placeholder with the website domain filled out in the basic detail step?